src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -461,6 +482,67 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
                   />
                 </Grid>
               )}
+            </Grid>
+            <>
+              {
+                isSingleBatch && !isOutplant && (
+                  // localRecord.batchWithdrawals.map((bw, index) => {
+                  //   return (
+                  <>
+                    <Grid display='flex' flexDirection={isMobile ? 'column' : 'row'}>
+                      <Grid item xs={gridSize()} sx={{ marginTop: theme.spacing(2) }} paddingRight={1}>
+                        <Textfield
+                          label={strings.NOT_READY_QUANTITY}
```

Comment:
the labels don't match the designs, also i think we need validation on these quantities.
the input heights are also off

Reply:
Because they are missing the *? I was waiting for Chudo to validate that, I don't think we need to input a ready and a not ready quantity

Reply:
yes, we need one or the other. i guess we can add the validation as a follow up.
 the other issues are mostly visual:
1) tooltip missing
2) input height smaller than other elements in the form
3) mobile view needs update